### PR TITLE
feat(sfcc): add SFCC output bus words

### PR DESCRIPTION
**Pull request by lukecologne**
## Summary of Changes

* Add some of the SFCC bus output words (see docs for added words)
   * Note that many bits are just hardcoded to true/false, since some features are not implemented yet.
* Modify the Flaps and Slats FPPU to surface angle conversions to better fit the real data. 
* Change the EWD over to use the IPPU/FPPU angles, and the SFCC bus output.
   * Note that the DMCs do not actually receive the SFCC bus directly, but via the SDAC. The words are just copied though,
     so the bit assignments etc. are the same. Also note that the DMCs normally use the IPPU angle from the FWC, and only switch over to SDAC FPPU bus data in case of FWC or IPPU failure.
   * As a result of the new flaps position computation via the IPPU angles, some of the graphical elements had to be slightly adjusted.

## Screenshots (if necessary)

Screenshots of the adjusted EWD flaps display: 
* Before: 
before_flaps
* After:
after_flaps

## Additional context

Discord username (if different from GitHub):

## Testing instructions

* Observe the flaps indicator on the EWD in all possible configurations, and make sure there are no issues/glitches (the computation method of the flaps indexes was significantly changed).
* Also, confirm that the flaps and slats physical impact is the same as before in all configurations

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by crocket63 — APPROVED** [review]
Looks good!

**Review by davidwalschots — DISMISSED** [review]
I do see some edits of tests, but it seems to me like quite some new functionality is added while tests are missing?

File: src/systems/a320_systems/src/hydraulic/mod.rs [hunk 1]
```diff
@@ -1218,18 +1218,16 @@ impl A320Hydraulic {
     const HIGH_PITCH_PTU_SOUND_DURATION: Duration = Duration::from_millis(3000);
 
     const FLAP_FFPU_TO_SURFACE_ANGLE_BREAKPTS: [f64; 12] = [
-        0., 65., 115., 120.53, 136., 145.5, 152., 165., 168.3, 179., 231.2, 251.97,
-    ];
-    const FLAP_FFPU_TO_SURFACE_ANGLE_DEGREES: [f64; 12] = [
-        0., 10.318, 18.2561, 19.134, 21.59, 23.098, 24.13, 26.196, 26.72, 28.42, 36.703, 40.,
+        0., 35.66, 69.32, 89.7, 105.29, 120.22, 145.51, 168.35, 189.87, 210.69, 231.25, 251.97,
     ];
+    const FLAP_FFPU_TO_SURFACE_ANGLE_DEGREES: [f64; 12] =
```
Comment (beheh) [inline src/systems/a320_systems/src/hydraulic/mod.rs:1223]: I think there might've been a typo here in the original code - surely this refers to the F**P**PU?

Comment (lukecologne) [reply src/systems/a320_systems/src/hydraulic/mod.rs:1223]: Oh yeah, it should be FPPU. I'll just correct it in this PR

**Comment by 2hwk** [comment]
@lukecologne needs testing instructions for QA

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #7221
Tier of Testing  : 1
Date             : 12/06/2022

Testing Process: 
Observe the flaps indicator on the EWD in all possible configurations, and make sure there are no issues/glitches. - **OK**
Also, confirm that the flaps and slats physical impact is the same as before in all configurations - **OK**

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
LGTM

**Comment by 2hwk** [comment]
To merge after 15/06

**Pull request merged**